profile/README.md

```diff
@@ -20,18 +18,29 @@ The Holoscan platform consists of:
 * **Deployment Software Stack**: OpenEmbedded/Yocto layer for optimized deployment of the Holoscan SDK on the Developer Kits
     * [OpenEmbedded/Yocto Build container](https://catalog.ngc.nvidia.com/orgs/nvidia/teams/clara-holoscan/containers/holoscan-oe-builder)
     * [OpenEmbedded/Yocto Recipes](https://github.com/nvidia-holoscan/meta-tegra-holoscan)
+* **Holoscan Sensor Bridge Software**: integrate the Holoscan Sensor Bridge Device in a Holoscan Application
+    * [Source code](https://github.com/nvidia-holoscan/holoscan-sensor-bridge)
+    * [Documentation](https://docs.nvidia.com/holoscan/sensor-bridge/latest)
+
+## Hardware
+
+### Developer Kits
 
-## Developer Kits
+* [**NVIDIA IGX Orin Developer Kit**](https://www.nvidia.com/en-us/edge-computing/products/igx/) ([User guide](https://github.com/nvidia-holoscan/holoscan-docs/blob/main/devkits/nvidia-igx-orin/nvidia_igx_orin_user_guide.md))
+* [**NVIDIA Clara AGX Developer Kit**](https://www.nvidia.com/en-gb/clara/intelligent-medical-instruments/) ([User guide](https://github.com/nvidia-holoscan/holoscan-docs/blob/main/devkits/clara-agx/clara_agx_user_guide.md))
+* [**NVIDIA Jetson Orin Developer Kits**](https://www.nvidia.com/en-us/autonomous-machines/embedded-systems/jetson-orin/) ([User Guide](https://developer.nvidia.com/embedded/learn/jetson-agx-orin-devkit-user-guide/index.html))
 
-* **[NVIDIA Clara AGX Developer Kit](https://www.nvidia.com/en-gb/clara/intelligent-medical-instruments/)**
-    * Jetson AGX Xavier™, RTX™ 6000 GPU, ConnectX® SmartNIC
-    * [User guide](https://github.com/nvidia-holoscan/holoscan-docs/blob/main/devkits/clara-agx/clara_agx_user_guide.md)
-* **[NVIDIA IGX Orin Developer Kit](https://www.nvidia.com/en-us/edge-computing/products/igx/)**
-    * Jetson AGX Orin™, RTX™ A6000 GPU, ConnectX® SmartNIC
-    * [User guide](https://github.com/nvidia-holoscan/holoscan-docs/blob/main/devkits/nvidia-igx-orin/nvidia_igx_orin_user_guide.md)
+### Sensors
+
+* [**Holoscan Sensor Bridge Reference Design**](https://www.latticesemi.com/products/developmentboardsandkits/certuspro-nx-sensor-to-ethernet-bridge-board): bring your own sensor, and leverage this FPGA design to achieve RDMA with the ConnectX NIC of a NVIDIA Developer Kit.
```

Comment:
Holoscan sensor bridge also works with non-RDMA configurations like AGX; might be worth saying something like _host systems with ConnectX can leverage that for RDMA..._.

Reply:
```suggestion
* [**Holoscan Sensor Bridge Reference Design**](https://www.latticesemi.com/products/developmentboardsandkits/certuspro-nx-sensor-to-ethernet-bridge-board): bring your own sensor, and leverage this FPGA design to send packets over Ethernet, including RDMA support when connected to the ConnectX NIC of a NVIDIA Developer Kit.
```